packages/component-library-react/src/index.ts

```diff
@@ -27,8 +27,8 @@ export type { BlockquoteProps } from './Blockquote';
 export { Blockquote } from './Blockquote';
 export type { BreadcrumbNavProps, BreadcrumbNavLinkProps, BreadcrumbNavSeparatorProps } from './BreadcrumbNav';
 export { BreadcrumbNav, BreadcrumbNavLink, BreadcrumbNavSeparator } from './BreadcrumbNav';
-export type { ButtonProps } from '@utrecht/button-react';
-export { Button, PrimaryActionButton, SecondaryActionButton, SubtleButton } from '@utrecht/button-react';
+export type { ButtonProps } from './Button';
+export { Button, PrimaryActionButton, SecondaryActionButton, SubtleButton } from './Button';
```

Comment:
Waarom fixt dit iets? `Button.tsx` exporteert toch precies dezelfde dingen?

Had je dit willen aanpassen in `css/`?

Reply:
Ja, ik wilde de css/ aanpassen, maar ik zie dat je dit vorige week al hebt opgelost in dit [commit](https://github.com/nl-design-system/utrecht/commit/ecaf6a6c12cb0c1a4d9708100ee5be9ef9a63774), dus ik ga deze PR sluiten.